### PR TITLE
Update 'Get message status' documentation (#NOT-222)

### PR DESCRIPTION
**Pull request by jonalport**
https://govausites.atlassian.net/browse/NOT-222

image

### CODE REVIEW

File: docs/src/code-examples/get-all-messages/python.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+lang: python
+---
+
+```python
+response = python_client.get_all_notifications(
+  templateType='sms',
```
Comment (dagoof) [inline docs/src/code-examples/get-all-messages/python.mdx:7]: template_type

File: docs/src/code-examples/get-all-messages/python.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  status='delivered',
+  reference='xxxxxxxxxxxx',
+  olderThanId='xxxxxxxxxxxx'
```
Comment (dagoof) [inline docs/src/code-examples/get-all-messages/python.mdx:10]: older_than_id

File: docs/src/code-examples/get-all-messages/python.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        <li><strong>failed</strong> - this will return all failure statuses permanent-failure, temporary-failure and technical-failure.</li>
+        <li><strong>permanent-failure</strong> - the provider was unable to deliver message, email does not exist; remove this recipient from your list.</li>
+        <li><strong>temporary-failure</strong> - the provider was unable to deliver message, email box was full; you can try to send the message again.</li>
+        <li><strong>technical-failure</strong> - Notify had a technical failure; you can try to send the message again.</li>
+      </ul>
+    </Parameter>
+    <Parameter name="reference">
+      Filter messages by reference supplied at time of sending.
+    </Parameter>
+    <Parameter name="olderThanId">
```
Comment (dagoof) [inline docs/src/code-examples/get-all-messages/python.mdx:39]: older_than_id

File: docs/src/code-examples/get-all-messages/python.mdx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  reference='xxxxxxxxxxxx',
+  olderThanId='xxxxxxxxxxxx'
+)
+```
+
+For each of the below arguments; if a falsy value is provided, the respective filter is not applied.
+
+<Panel label={<H3 appearAs={H5}>Arguments</H3>}>
+  <Parameters>
+    <Parameter name="templateType">
```
Comment (dagoof) [inline docs/src/code-examples/get-all-messages/python.mdx:18]: template_type

File: docs/src/code-examples/get-all-messages/python.mdx (hunk 3 above)
Comment (jonalport) [reply docs/src/code-examples/get-all-messages/python.mdx:39]: Ah the underscores!

**Comment by jonalport** [comment]
Cheers, will update

**Pull request merged**